modules/openapi-generator/src/main/resources/go/model_simple.mustache

```diff
@@ -168,6 +168,18 @@ func (o *{{classname}}) Set{{name}}(v {{vendorExtensions.x-go-base-type}}) {
 {{/isNullable}}
 }
 
+{{^isReadOnly}}
+{{#defaultValue}}
+{{#initRequiredVars}}
+func (o *{{classname}}) GetDefault{{baseName}}() interface{}  {
+	var {{baseName}} {{{dataType}}} = {{{defaultValue}}}
```

Comment:
i don't think we need an additional variable to hold the default value which can be used directly in the line below